vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj

```diff
@@ -387,6 +390,95 @@
   <ItemGroup>
     <ClCompile Include="ABICxxModule.cpp" />
     <ClCompile Include="ABIViewManager.cpp" />
+    <ClCompile Include="$(ReactNativeDir)\ReactCommon\react\config\ReactNativeConfig.cpp" Condition="'$(UseFabric)' == 'true'" />
```

Comment:
nit, you could create a separate `ItemGroup` with the  `UseFabric` condition in it and move all these files underneath it so you don't have to repeat yourself